kafka-rest/src/main/java/io/confluent/kafkarest/v2/KafkaConsumerManager.java

```diff
@@ -129,7 +126,7 @@ public void rejectedExecution(Runnable r, ThreadPoolExecutor executor) {
                   + "The thread pool must be satured or shutiing down", r);
```

Comment:
replace this single log message (which has a typo and is not well worded) with a different one for each case

Reply:
maybe:

`The runnable {} was rejected execution because the thread pool is saturated. Retrying in {}ms`.

`The runnable {} was rejected execution because the thread pool is terminating`.

Reply:
Done.